CHANGELOG.md

```diff
@@ -6,22 +6,37 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/ASFHyP3/hyp3-lib/compare/v1.4.1...develop)
+## [1.5.0](https://github.com/ASFHyP3/hyp3-lib/compare/v1.5.0...1.4.1)
 
-### changed
+### Added
+* Documented contribution guidelines in CONTRIBUTING.md
+* `get_dem.get_dem` has a new `dem_type` parameter with options of `utm` (default), `latlon`, and `isce`
+
+### Changed
 * Requires python >= 3.6
+* `S1_OPOD_vec` calls to apply S1 state vectors to SLC products are now logged. Logging was already in place for GRD
+  products.
+* Configuration of the `hyp3lib.get_dem` module has seen the following changes:
+  * The module will attempt to load `~/.hyp3/get_dem.cfg`; if not found it will load `hyp3lib/etc/config/get_dem.cfg`.
+  * `get_dem.cfg` must be a space-delimited file of the form `<dem_name> <location> <epsg_code>`
+  * `<location>` may be either a local path (e.g. `/foo/bar/`) or a url prefix (e.g. `https://foo.com/bar/`).
+    S3 prefixes are no longer supported (e.g. `s3://foo/bar/`).
+  * Shapefiles describing each DEM's coverage are no longer packaged with hyp3lib.  They must now be provided at
+    `<location>/coverage/<dem_name>_coverage.shp`.
```

Comment:
Might want to mention that these are lowercase names:

https://github.com/ASFHyP3/hyp3-lib/blob/develop/hyp3lib/get_dem.py#L55

Reply:
OR we make the config file and shape files match...

Reply:
Ehh, punt this to the dem/config rewrite